common/trans.c

```diff
@@ -831,6 +831,26 @@ trans_listen_address(struct trans *self, char *port, const char *address)
             }
         }
     }
+#if defined(XRDP_ENABLE_VSOCK)
+    else if (self->mode == TRANS_MODE_VSOCK) /* vsock socket */
+    {
+        self->sck = g_vsock_socket();
+        if (self->sck < 0)
+            return 1;
```

Comment:
We always use braces for if statements. See `coding_style.md`.

Reply:
Good catch. Updated the other locations that I based this off of.